### ISSUE TITLE
resizeFramebufferInfo and createFramebufferInfo

### ISSUE THREAD

**Issue by kaktus40** [greggman/twgl.js#67]
Hello,
first thanks for your work on twgl. As a beginner in webgl its a great help.
Following your documentation, I try to create a frameBuffer with a certain dimension and I'd like to resizing the created frameBuffer. In chromium, the resizing action generates the following error:
```
[.Offscreen-For-WebGL-0x23fce9081a00]GL ERROR :GL_INVALID_OPERATION : glTexImage2D: invalid internalformat/format/type combination GL_RGB/GL_RGB/GL_FLOAT
```
my code is:
```
const gl=canvas.getContext('webgl2', { antialias: false });
twgl.addExtensionsToContext(gl);
gl.getExtension('EXT_color_buffer_float');
gl.getExtension('OES_texture_float_linear');
gl.getExtension('OES_texture_float');
gl.getExtension('WEBGL_color_buffer_float');
.
.
.
const attachments = [{ format: gl.RGB, type: gl.FLOAT,internalFormat:gl.RGB32F, minMag: gl.NEAREST, wrap: gl.CLAMP_TO_EDGE }];
const fbi = twgl.createFramebufferInfo(gl, attachments, width,height);
twgl.bindFramebufferInfo(gl)
.
.
.
 twgl.resizeFramebufferInfo(gl, fbi,attachments, newWidth,newHeight);
```
I don't understand that ``createFramebufferInfo`` works and with the same attachments attributes,  ``resizeFramebufferInfo`` doesn't work....

**Comment by greggman** [comment]
It was a bug. Sorry about that. Fixed in 4.1.0

https://github.com/greggman/twgl.js/commit/2c8a57560e138fc133e7c8cf6c44f39e941a03ea

**Comment by kaktus40** [comment]
Thanks, it works now.
